g/go-oidc/go-oidc_ubi_8.5.sh

```diff
@@ -2,13 +2,13 @@
 # -----------------------------------------------------------------------------
 #
 # Package	: go-oidc
-# Version	: v2.1.0
+# Version	: v2.1.0, v0.0.0-20180117170138-065b426bd416
```

Comment:
since there are changes in the script, have we confirmed that it still works for v2.1.0 as well?

Reply:
@seth-priya - Existing script is failing for both v2.1.0 & 0.0.0-20180117170138-065b426bd416, so done required changes for script which will pass for both versions.